vllm/model_executor/layers/sparsity/semi_structured_sparse_w16a16.py

```diff
@@ -0,0 +1,49 @@
+import torch
+
+from typing import Any, Dict, List, Type
+from magic_wand import CompressedStorageFormat
+from vllm.model_executor.layers.sparsity.base_config import SparsityConfig
+from .sparse_w16a16_linear_method import SparseW16A16LinearMethod
+from magic_wand import (
+    CompressedStorageFormat,
+    SparseSemiStructuredStorageFormat
+)
+
+class SemiStructuredSparseW16A16Config(SparsityConfig):
+    """Config class for SemiStructuredSparseW16A16.
+    """
+
+    def __init__(self) -> None:
+        # TODO: Add new configs here
+        pass
+
+    def __repr__(self) -> str:
+        return "SemiStructuredSparseW16A16Config()"
+
+    @classmethod
+    def get_storage_format_cls(cls) -> Type[CompressedStorageFormat]:
+        return SparseSemiStructuredStorageFormat
+
+    @classmethod
+    def get_name(cls) -> str:
+        return "semi_structured_sparse_w16a16"
+
+    @classmethod
+    def get_supported_act_dtypes(cls) -> List[torch.dtype]:
+        return [torch.half]
+
+    @classmethod
+    def get_min_capability(cls) -> int:
+        # TODO: Update after checks on more GPUs
+        return 80
+
+    @classmethod
+    def get_config_filenames(cls) -> List[str]:
+        return ["sparsity_config.json"]
```

Comment:
is this required? i don't know if we decided we will store information here or in the config.json yet? @rib-2 ?

Reply:
I am mirroring the structure of @alexm-nm 's sparsity config in [here](https://github.com/neuralmagic/neuralmagic-vllm/blob/dbb125bb4dfe577a3068eacd56b2c921bbca6549/vllm/model_executor/layers/sparsity/sparse_w16a16.py#L36-L40) :

```python
@classmethod
def get_config_filenames(cls) -> List[str]:
    return ["sparsity_config.json"]
```

Reply:
Just checked in with Robert, we haven't decided yet on how to handle the config file so sparsity_config.json is fine for now